src/remediations/controller.read.js

```diff
@@ -301,8 +301,11 @@ exports.playbook = errors.async(async function (req, res) {
     const tenant_org_id = req.identity.org_id;
     const creator = cert_auth ? null : req.user.username;
 
+    const USE_CACHE = true;
+    const EXCLUDE_RESOLVED_COUNT = false;
```

Comment:
Why uppercase?

Reply:
Convention is to use uppercase for things that are static / hard-coded.

Reply:
In that case, those should be moved to the top of the file. `const` keywords denotes a constant already.